src/main/cc/proxy_client/proxy_client.cc

```diff
@@ -429,7 +445,7 @@ int ComputeInputs(int argc, char** argv, const char** env, const string& cwd, co
 
 int CreateRunRequest(int argc, char** argv, const char** env,
                      const string& cmd_id, RunRequest* req,
-                     bool* is_compile, bool* is_javac) {
+                     bool* is_compile, bool* is_javac, bool* is_link) {
```

Comment:
We should probably make this an enum like Ramy did in ninja.

Reply:
Can I do that separately? I would like to merge this and get story points ;)

Reply:
Oh, totally, I didn't mean in this PR! I wish there was a way to "mark as resolved" when I comment, like in critique.

Reply:
Great thanks, I'll followup with a separate CL for it.